nexuse2e-webapp/src/main/frontend/src/app/pipes/date-range.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { DateRange } from "../types";
+
+@Pipe({
+  name: "dateRange",
+  pure: true,
+})
+export class DateRangePipe implements PipeTransform {
+  transform(value: unknown): DateRange {
+    return value as DateRange;
```

Comment:
Is a logging for the case the value is not a DateRange useful?

Reply:
Done